client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java

```diff
@@ -86,6 +86,31 @@ public class CommonItemConfig {
     @ConfigItem(name = "mutiny")
     public Optional<Boolean> supportMutiny;
 
+    /**
+     * Handles the return type for each operation, depending on the configuration.
+     * The following cases are supported:
+     * <p>
+     * 1. If `mutiny` is enabled and the operation ID is specified to return `Multi`:
+     * - The return type will be wrapped in {@link io.smallrye.mutiny.Multi}.
+     * - If `return-response` is enabled, the return type will be {@link io.smallrye.mutiny.Multi<jakarta.ws.rs.core.Response>}.
+     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Multi<jakarta.ws.rs.core.Response>}.
+     * - Otherwise, it will return {@link io.smallrye.mutiny.Multi<returnType>}.
+     * <p>
+     * 2. If `mutiny` is enabled and the operation ID is specified to return `Uni`:
+     * - The return type will be wrapped in {@link io.smallrye.mutiny.Uni}.
+     * - If `return-response` is enabled, the return type will be {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
+     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
+     * - Otherwise, it will return {@link io.smallrye.mutiny.Uni<returnType>}.
+     * <p>
+     * 3. If `mutiny` is enabled but no specific operation ID is configured for `Multi` or `Uni`:
+     * - The return type defaults to `Uni`.
+     * - If `return-response` is enabled, the return type will be {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
+     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
+     * - Otherwise, it will return {@link io.smallrye.mutiny.Uni<returnType>}`.
+     */
```

Comment:
Javadoc doesn't support Markdown. We need to use `{@code mutiny}` instead of \`mutini\`, for example.

```suggestion
    /**
     * Handles the return type for each operation, depending on the configuration.
     * The following cases are supported:
     * <p>
     * 1. If {@code mutiny} is enabled and the operation ID is specified to return {@code Multi}:
     * - The return type will be wrapped in {@link io.smallrye.mutiny.Multi}.
     * - If {@code return-response} is enabled, the return type will be {@link io.smallrye.mutiny.Multi<jakarta.ws.rs.core.Response>}.
     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Multi<jakarta.ws.rs.core.Response>}.
     * - Otherwise, it will return {@link io.smallrye.mutiny.Multi<returnType>}.
     * <p>
     * 2. If {@code mutiny} is enabled and the operation ID is specified to return {@code Uni}:
     * - The return type will be wrapped in {@link io.smallrye.mutiny.Uni}.
     * - If {@code return-response} is enabled, the return type will be {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
     * - Otherwise, it will return {@link io.smallrye.mutiny.Uni<returnType>}.
     * <p>
     * 3. If {@code mutiny} is enabled but no specific operation ID is configured for {@code Multi} or {@code Uni}:
     * - The return type defaults to {@code Uni}.
     * - If {@code return-response} is enabled, the return type will be {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
     * - If the operation has a void return type, it will return {@link io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response>}.
     * - Otherwise, it will return {@link io.smallrye.mutiny.Uni<returnType>}`.
     */
```